packages/node_modules/@webex/webex-core/src/lib/services/services.js

```diff
@@ -189,7 +190,7 @@ const Services = WebexPlugin.extend({
       formattedQuery = {};
 
       if (queryKey === 'email') {
```

Comment:
Would a better fix for this be:

```suggestion
      if (queryKey === 'email' && query.email) {
```

@InteractiveTimmy @chrisadubois

Reply:
yes but...optional chaining is finally in the SDK haha

Reply:
I'll change it